src/openms/source/FORMAT/HANDLERS/MzXMLHandler.cpp

```diff
@@ -988,9 +1039,19 @@ namespace OpenMS
         }
       }
 
-      os << "\t</msRun>\n"
-         << "\t<indexOffset>0</indexOffset>\n"
-         << "</mzXML>\n";
+      os << "\t</msRun>\n";
+
+      // create scan index (does not take a lot of space and is required for MaxQuant)
```

Comment:
can you make the index optional? I dont think its a problem but 

- we do have the `options_.getWriteIndex()` flag
- there may be software out there that cant deal with the index, so it would be frustrating to not be able to remove it